src/Tribe/Hooks.php

```diff
@@ -86,20 +88,55 @@ protected function add_actions() {
 	/**
 	 * Adds the filters required by the plugin.
 	 *
-	 * @since __TRIBE_VERSION__
+	 * @since 1.0.0
```

Comment:
Needs to add a since comment:
```suggestion
	 * @since 1.0.0
	 * @since 1.5.0 Added filter to inject a trash option to the bulk actions.
```